py/picca/io.py

```diff
@@ -791,48 +599,31 @@ def read_from_mock_1d(filename,
         cont = hdu["continuum"][:]
         mef = mean_flux_transmission * cont
         pix_data.append(
-            Forest(log_lambda, flux, ivar, t, r, d, z, p, m, f, order,
+            Forest(log_lambda, flux, ivar, 
+                   entry['THING_ID'], 
+                   entry['RA'], entry['DEC'], entry['Z'], 
+                   entry['PLATE'], entry['MJD'], entry['FIBERID'],
                    exposures_diff, reso, mef))
 
     hdul.close()
 
     return pix_data
 
-
+#-- Is anyone using this? 
```

Comment:
I don't think so, but I would keep it for a bit longer. Maybe we can add a DeprecationWarning if this function is used

Reply:
I don't know if somebody uses it, but agree with Ignasi about the warning

Reply:
added a warning